spec/syntax.md

```diff
@@ -915,16 +930,17 @@ in this release.
 variable   = "$" name
 option     = identifier [s] "=" [s] (literal / variable)
 
-identifier = [namespace ":"] name
-namespace  = name
-name       = name-start *name-char
-name-start = ALPHA / "_"
-           / %xC0-D6 / %xD8-F6 / %xF8-2FF
-           / %x370-37D / %x37F-1FFF / %x200C-200D
-           / %x2070-218F / %x2C00-2FEF / %x3001-D7FF
-           / %xF900-FDCF / %xFDF0-FFFC / %x10000-EFFFF
-name-char  = name-start / DIGIT / "-" / "."
-           / %xB7 / %x300-36F / %x203F-2040
+identifier       = [namespace ":"] name
+namespace        = name
+name             = name-start *name-char
```

Comment:
One option would be leaving out the `"."` from `name-char`, dropping `name-char-no-dot`, and including here something like:

```suggestion
name             = name-start *(name-char / ".")
```

That would somewhat imply our intent for `.` to be used as a separator between name parts without actually making it special in the syntax.

Please take this as purely a suggestion, I'm much more interested in the actual syntax rather than the rules we use to describe it.

Reply:
That seems reasonable.